watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/tests/test_dye_desalination.py

```diff
@@ -49,53 +50,82 @@ def test_set_operating_conditions(self, system_frame):
+        assert pytest.approx(2, rel=1e-5) == value(m.fs.feed.conc_mass_comp[0, "tds"])
+
+        assert pytest.approx(0.2, rel=1e-5) == value(m.fs.feed.conc_mass_comp[0, "dye"])
+
+        # test wwtp
+        assert pytest.approx(0.14, rel=1e-5) == value(
+            m.fs.pretreatment.wwtp.energy_electric_flow_vol_inlet
         )
 
-        assert pytest.approx(0.155556, rel=1e-3) == value(
-            m.fs.feed.flow_mass_comp[0, "tds"]
+        # test pump block
+        assert pytest.approx(7, rel=1e-5) == value(
+            m.fs.dye_separation.P1.applied_pressure[0]
         )
 
-        # test nanofiltration block
-        assert pytest.approx(316.219, rel=1e-3) == value(
+        # test nanofiltration
+        assert pytest.approx(315.9843, rel=1e-5) == value(
             m.fs.dye_separation.nanofiltration.area
         )
 
-        # test pump block
-        assert pytest.approx(7, rel=1e-6) == value(
-            m.fs.dye_separation.P1.applied_pressure[0]
+        # check products
+        assert pytest.approx(0, rel=1e-6) == value(
+            m.fs.wwt_retentate.flow_mass_comp[0, "dye"]
+        )
+        assert pytest.approx(0.026809, rel=1e-3) == value(
+            m.fs.concentrated_dye.flow_mass_comp[0, "tds"]
+        )
+
+        assert pytest.approx(0.965, rel=1e-5) == value(
+            m.fs.permeate.flow_mass_phase_comp[0, "Liq", "H2O"]
+        )
+
+        assert pytest.approx(0.035, rel=1e-5) == value(
+            m.fs.brine.flow_mass_phase_comp[0, "Liq", "TDS"]
         )
 
     @pytest.mark.component
     def test_solve(self, system_frame):
         m = system_frame
+
         results = solve(m)
 
-        # check each product
-        assert pytest.approx(64.4135, rel=1e-6) == value(
-            m.fs.permeate.flow_mass_comp[0, "H2O"]
+        # check products
+        assert pytest.approx(0.02894, rel=1e-3) == value(
+            m.fs.wwt_retentate.flow_mass_comp[0, "tds"]
+        )
+        assert pytest.approx(13.1931, rel=1e-3) == value(
+            m.fs.concentrated_dye.flow_mass_comp[0, "H2O"]
+        )
+
+        assert pytest.approx(32.2185, rel=1e-5) == value(
+            m.fs.permeate.flow_mass_phase_comp[0, "Liq", "H2O"]
         )
 
-        assert pytest.approx(0.015426, rel=1e-3) == value(
-            m.fs.dye_retentate.flow_mass_comp[0, "dye"]
+        assert pytest.approx(32.1950, rel=1e-5) == value(
+            m.fs.brine.flow_mass_phase_comp[0, "Liq", "H2O"]
         )
 
     @pytest.mark.component
     def test_costing(self, system_frame):
         m = system_frame
 
         add_costing(m)
+        initialize_costing(m)
+        optimize_operation(m)
+
         results = solve(m)
         assert_optimal_termination(results)
 
-        # check values
-        assert pytest.approx(19.626110, rel=1e-3) == value(m.fs.LCOT)
+        # check costing
+        assert pytest.approx(26.933615, rel=1e-3) == value(m.fs.LCOW)
+        assert pytest.approx(19.781864, rel=1e-3) == value(m.fs.LCOT)
```

Comment:
Any known reasons why the LCOT changed?

Reply:
This is a product of the naming being changed. So this test file was originally called `test_dye_desalination_withRO.py`, but since we combined the two flowsheets, I decided to drop the `withRO`, which just so happens to be the name of the test file that I deleted (the test file for the `dye_desalination.py` flowsheet, which did not include RO).

TLDR: these values didn't actually change. See https://github.com/watertap-org/watertap/blob/main/watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/tests/test_dye_desalination_withRO.py#L127